pyxdsm/matrix_eqn.py

```diff
@@ -1,8 +1,9 @@
 import os
 import subprocess
-from collections import namedtuple
-import numpy as np
+from typing import Dict, List, Optional, Tuple, Union
```

Comment:
`Dict`, `List`, and `Tuple` are deprecated since python 3.9 in favour of builtins `dict` etc. Unions can also be written via `|` starting with 3.10.